src/test/java/com/training/services/tests/VoyageCurrentSailValidateTest.java

```diff
@@ -43,4 +44,26 @@ public void testVoyageCurrentSailDateValidate(){
                 .matches("2020[0,1][0-2][0-3][0-9]")).
                 isTrue().as("is not in  the year 2020");
     }
+
+    /**
+     * testVoyageSailDurationValidate:validating sail duration should br less than 10
+     */
+    @Test
+    public void testVoyageSailDurationValidate() {
+        Response voyageResponse =
+                new RestEngine().getResponseGet(map.get("url_base")
+                                +"/en/royal/mobile/v3/ships/al/voyages"
+                         ,headerMap)
+                        .as(Response.class);
+        logger.info("status->" + voyageResponse.getStatus());
+        logger.info("totalVoyagesCount->"+voyageResponse.getPayload().getVoyages().size());
+        Assertions.assertThat(voyageResponse.getStatus()).isEqualTo("200")
+                .as(" status is not 200");
+        for (int i = 0; i < voyageResponse.getPayload().getVoyages().size(); i++) {
```

Comment:
Lets create a variable 
`List<Voyages> voyages = voyageResponse.getPayload().getVoyages();`
and then use this list in your for loop, would make it more readable